QueueMessageSender/QueueMessageSender/Models/RegistrationModel.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace QueueMessageSender.Models
+{
+    /// <summary>
+    /// Credential model
+    /// </summary>
+    public class RegistrationModel
+    {
+        [Required]
+        public string Username { get; set; }
+
+        [Required]
+        [DataType(DataType.Password)]
+        public string Password { get; set; }
+
+        [Required]
+        public string Role { get; set; }
```

Comment:
Why Role string value is used instead of id / Key? It is ok to use string value because most probably it will be unique but in this case if could be made as primary key.